### PR TITLE
Change README to tell users to use gradle wrapper

### PR DESCRIPTION
**Pull request by MrJeremyFisher**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ This monorepo will eventually contain all civ projects and development
 A docker compose stack is provided to help test containers built from
 this repo. To start the stack, run the following commands:
 
-1. `gradle :ansible:build`
+1. `./gradlew :ansible:build`
```
Comment (AngrySoundTech) [inline README.md:13]: This is only the case for linux/unix, should be `.\gradlew.bat build` for PowerShell and I believe `gradlew.bat build` for cmd on windows?

**Review by AngrySoundTech — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (MrJeremyFisher) [reply README.md]: Sorry, missed this. Powershell and CMD on Windows 10 both accept `.\gradlew`

**Comment by AngrySoundTech** [comment]
`.\gradlew :ansible:build` doesn't always work on windows, only in powershell which is isn't fully POSIX compliant. should use `gradlew.bat`

**Comment by MrJeremyFisher** [comment]
Does it not work for you in cmd? 
image

**Comment by okx-code** [comment]
gradlew.bat would be preferable here

**Review by AngrySoundTech — APPROVED** [review]

**Pull request merged**